ui/src/variables/components/VariableImportOverlay.tsx

```diff
@@ -40,23 +32,19 @@ class VariableImportOverlay extends PureComponent<Props> {
   }
 
   private handleImportVariable = async (
-    uploadContent: string,
-    orgID: string
+    uploadContent: string
   ): Promise<void> => {
-    const {createVariableFromTemplate, getVariables} = this.props
+    const {createVariableFromTemplate} = this.props
 
     const template = JSON.parse(uploadContent)
-    await createVariableFromTemplate(template, orgID)
-
-    await getVariables()
```

Comment:
is this no longer required to be able to see all imported variables?

Reply:
with variable templates including other variables, it's probably better to getVariables.